src/process/executable_path.cr

```diff
@@ -28,13 +28,45 @@ class Process
     end
   end
 
+  private def self.is_executable_file?(path)
+    unless File.info?(path, follow_symlinks: true).try &.file?
+      return false
+    end
+    {% if flag?(:win32) %}
+      true
```

Comment:
I would add a comment here that there is nothing else to be done.
Otherwise it seems like something is missing for windows, when it's not, for once! right?